packages/gcc8.rb

```diff
@@ -20,11 +20,7 @@ class Gcc8 < Package
      x86_64: '6d982f3c35acd3a738e176a3f3b8c4ea093c7be86882153a7711841cf0c80737',
   })
 
-  depends_on 'unzip' => :build
-  depends_on 'gawk' => :build
   depends_on 'dejagnu' => :build # for test
-  depends_on 'gcc7' => :build # gcc version 7.4.0
```

Comment:
You won't get past the configure script without a compiler.

Reply:
We're going to add binaries before we merge this, just like any core package. If you do this, there's a dependency loop. Also, Chromebrew always installs a compiler during installation, so you should have no problem running `crew build gcc...`

Reply:
One of the main reasons I did this is because I don't want a compiler to depend on another compiler. If we did that, it could break stuff, including having two conflicting compilers installed on one system. If we have two conflicting compilers on one system, there would be library mismatches. Also, if we removed one compiler, half of the other would also be removed. We really need to figure out a way to handle file collisions.